IFC4x3/Properties/t/TargetPerformance_3YBpMKO1H0zhLW9cAjhxfY/DocProperty.xml

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TargetPerformance_3YBpMKO1H0zhLW9cAjhxfY" Name="TargetPerformance" UniqueId="e22f3594-6014-40f6-b560-2662adafba62" PrimaryDataType="IfcCountMeasure" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="TargetPerformance" UniqueId="e22f3594-6014-40f6-b560-2662adafba62" PrimaryDataType="IfcCountMeasure" />
```

Comment:
Found an orphan. :)

Reply:
I think at ifc2x2->ifc2x3 some psets were deleted (without consultation). TargetPerfrormance was in one Pset_perfromance or pset_PAMPER (????). Should be reinstated now that we are considering life-cycle issues more fully.

Reply:
Hi nick, it is still in just an encoding issue in IfcDoc! :)